client/src/components/Allocations/Allocations.scss

```diff
@@ -106,6 +106,11 @@
     font-size: 0.875rem; /* 14px (approved deviation from design) */
   }
 }
+/* Selector is long so it is specific enough to override without !important */
+/* NOTE: The length of selector helps devs find the source of the overwrite */
+.allocations-table.InfiniteScrollTable tbody tr:not(.-status) td {
+  vertical-align: top;
+}
```

Comment:
**Feedback welcome.** I did not use `!important`, because I avoid [using `!important` reactively](https://confluence.tacc.utexas.edu/display/UP/UI+-+CSS+Style+Guide+-+Specificity#UICSSStyleGuideSpecificity-Reactive). **But** this stylesheet (and InfiniteScrollTable's) need love[^1] and I don't have bandwidth to give it yet, so I might as well just throw an `!important` in and remove the comments. Anyway, if this diff looks more crazy than helpful, call me out, and I will just use `!important`.

[^1]: I've tried giving love to both Allocations and InfiniteScrollTable CSS, but always in the middle of different large tickets, and the changes were out of scope, and kept forcing me to fix code higher up. I just couldn't isolate the fix. Maybe the new Core-Components repo (coming soon) is my chance to create the one TACC Table component. 🤩